V1.2 + Internationalization + update to purecss 0.6

Sebseb01 opened the pull request:
# Chang log
- Change version number to 1.2
- Simply-pure is now fully translatable! (i think all string use gettext)
- Propose a french version of Simply-Pure
- I have update to PureCSS 0.6 (need some adjustement in html+css)
- Remove old unused code
# ToDo
- Menu use a div for pure-button (I have not found a way to pure-button in the link) ... need to fix it

Commit:
1st version translatable (maybe fully)

Commit:
With all file ;-)

Sebseb01 committed:
Merge remote-tracking branch 'upstream/master'

Conflicts:
	content-loop.php

Sebseb01 committed:
change to v1.2

Sebseb01 committed:
Add pagination translation
And new version for translating metadata

Sebseb01 committed:
Add translation for archive title

Sebseb01 committed:
Update to purecss 0.6 + more traslation comment forms

Sebseb01 committed:
very small bug fix

Sebseb01 committed:
FIX : Menu not centred

Sebseb01 committed:
FIX : Menu is not centred

Sebseb01 committed:
Add compatibiliy styling for other forms (not only comments)

Sebseb01 committed:
Better translate for comment forms

Sebseb01 committed:
Remove some html comment for my website

riteshsanap commented on functions.php:
```diff
@@ -1,284 +1,290 @@
\ No newline at end of file
+<?php 
+
+load_theme_textdomain('purecss', get_template_directory() . '/languages');
```
No need to use load_theme_textdomain() 

instead check here 
https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/

Sebseb01 committed:
Integrate change of v1.1.1
Merge remote-tracking branch 'upstream/master'

Conflicts:
	comments.php
	page.php
	style.css
	style.scss

Sebseb01 committed:
Correct merge of changelog

Sebseb01 committed:
remove @import ... do job with wp API

Sebseb01 committed:
Translate next/prev page
remove strange file Thumbs.db

Sebseb01 commented:
+pure_meta_data is uncommented in last commit (before your comment)
+I have integrate your change in version 1.1.1 !
+And for @import, i have remove them and re-add with wp_enqueue_style, it better for lisibilty and simplest way for upgrade pure-css (some other plugin can optimize wp_enqueue_style for join them in only one file)

thanks for your comments

riteshsanap commented:
I have updated your code and made it into a branch : https://github.com/riteshsanap/Simply-Pure/tree/sebseb01-master 

I will merge it with master after fixing some more stuff and while releasing the next version, which might take a few weeks
